pkg/kubectl/cmd/create.go

```diff
@@ -65,6 +65,7 @@ func NewCmdCreate(f cmdutil.Factory, out, errOut io.Writer) *cobra.Command {
 		Example: createExample,
 		Run: func(cmd *cobra.Command, args []string) {
 			if cmdutil.IsFilenameSliceEmpty(options.FilenameOptions.Filenames) {
+				templates.UseMainUsageTemplatesWithoutFlags(cmd)
```

Comment:
this mean we need set this func in ```kubectl set```,```kubectl config``` ? can we set this into ```DefaultSubCommandRun```?

Reply:
thanks for the comments.

`DefaultSubCommandRun` is used by following commands

> kubectl auth
> kubectl config
> kubectl create
> kubectl create secret
> kubectl create service
> kubectl plugin
> kubectl rollout
> kubectl set
> kubectl top

I think we can set the func into `DefaultSubCommandRun`, and the options will be hidden when simply run above commands(no -h or help) if they print options before.